storage/client/client.go

```diff
@@ -17,6 +17,7 @@ import (
 	staking "github.com/oasisprotocol/oasis-core/go/staking/api"
 	oasisConfig "github.com/oasisprotocol/oasis-sdk/client-sdk/go/config"
 
+	"github.com/oasisprotocol/oasis-indexer/analyzer/runtime"
```

Comment:
this goes the wrong way. analyzer should depend on storage instead. moving `EVMEthAddrFromPreimage` to be with `EthChecksumAddrFromPreimage` here